dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/document/visibility/Authorizations.java

```diff
@@ -0,0 +1,369 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rya.mongodb.document.visibility;
+
+import static com.google.common.base.Charsets.UTF_8;
+
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+import com.google.common.base.Charsets;
+
+/**
+ * A collection of authorization strings.
+ */
+public class Authorizations implements Iterable<byte[]>, Serializable, AuthorizationContainer {
+
+  private static final long serialVersionUID = 1L;
+
+  private final Set<ByteSequence> auths = new HashSet<ByteSequence>();
+  private final List<byte[]> authsList = new ArrayList<byte[]>(); // sorted order
+
+  /**
+   * An empty set of authorizations.
+   */
+  public static final Authorizations EMPTY = new Authorizations();
```

Comment:
same as above.  The constant class had a empty document visibility.   It also referenced this constant to let be known it was for ALL_AUTHORIZATIONS like in Accumulo Rya.